packages/firebase_performance/firebase_performance/example/lib/main.dart

```diff
@@ -7,15 +7,16 @@
 
 import 'dart:async';
 
-import 'package:firebase_core/firebase_core.dart';
 import 'package:firebase_performance/firebase_performance.dart';
 import 'package:flutter/material.dart';
 import 'package:http/http.dart';
 import 'package:pedantic/pedantic.dart';
 
 Future<void> main() async {
   WidgetsFlutterBinding.ensureInitialized();
-  await Firebase.initializeApp();
+  // Firebase Performance Monitoring uses default app so no need to initialize
+  // it here.
+  // await Firebase.initializeApp();
```

Comment:
@russellwheatley could you confirm if this? Note performance does not yet support multi-app so it always uses the default app. In that case can we skip the `iniaializeApp()` call?

Reply:
It seems to be confirmed here:
https://github.com/FirebaseExtended/flutterfire/blob/master/packages/firebase_core/firebase_core/lib/src/firebase.dart#L36